releases/patch-releases.md

```diff
@@ -59,47 +59,57 @@ releases may also occur in between these.
 
 ### 1.19
 
-Next patch release is **1.19.1**.
+Next patch release is **1.19.3**
 
-| Patch Release | Cherry-picks deadline | Target date |
-| --- | --- | --- |
-| 1.19.2 | 2020-09-11 | 2020-09-16 |
-| 1.19.1 | 2020-09-04 | 2020-09-09 |
+End of Life for **1.19** is **2021-09-30**
+
+| PATCH RELEASE | CHERRY PICK DEADLINE | TARGET DATE |
+|--- |--- |--- |
+| 1.19.3        | 2020-10-09           | 2020-10-14  |
+| 1.19.2        | 2020-09-11           | 2020-09-16  |
+| 1.19.1        | 2020-09-04           | 2020-09-09  |
 
 ### 1.18
 
-Next patch release is **1.18.9**.
+Next patch release is **1.18.10**
 
-| Patch Release | Cherry-picks deadline | Target date |
-| --- | --- | --- |
-| 1.18.9 | 2020-09-11 | 2020-09-16 |
-| 1.18.8 | N/A | 2020-08-13 |
-| 1.18.7 | 2020-08-07 | 2020-08-12 |
-| 1.18.6 | 2020-07-10 | 2020-07-15 |
-| 1.18.5 | 2020-06-25 | 2020-06-26 |
-| 1.18.4 | 2020-06-12 | 2020-06-17 |
-| 1.18.3 | 2020-05-15 | 2020-05-20 |
-| 1.18.2 | 2020-04-13 | 2020-04-16 |
-| 1.18.1 | 2020-04-06 | 2020-04-08 |
+End of Life for **1.18** is **TBD**
+
+| PATCH RELEASE | CHERRY PICK DEADLINE | TARGET DATE |
+|--- |--- |--- |
+| 1.18.10       | 2020-10-09           | 2020-10-14  |
```

Comment:
Was this an automated formatting change from your editor, eg: in response to the column headers getting wider?  If so could that be disabled in your editor and the proposed formatting changes put in a separate PR?

I'm partly ok with the change in formatting as it feels more readable, but that should be a separate patch and we'd need to have some enforcing of the format so that somebody else's tool doesn't remove all the trailing spaces.

Or...was this partly the markdown generated from the yaml file plus some trailing space deleted from the 1.17 section cells?  If so...for a separate PR we should insert a non-printing comment in the document here saying "don't edit this file's markdown tables directly rather edit the yaml, run the generator, move the generated output into this file" (or better really commit to using the generator and update the generator to do that edit).

The goal of going into all the detail around formatting is to achieve a point where a PR like this is trivially and quickly reviewed because it has a +6 -3 diffstat instead of +60 −41 diffstat.

Reply:
I did some formating, to make nicer, but i can follow the one that already exists

Reply:
also i can use the one that the tool outputs